main.go

```diff
@@ -111,23 +110,22 @@ func launchServer() error {
 		return fmt.Errorf("server routing configuration failed: %w", err)
 	}
 
-	orgProfileURL := cfg.Server.OrgProfileURL
+	orgProfileLocation := cfg.Server.OrgProfile
 
 	// Start Goroutine to refresh the organization profile every 5 minutes
-	if orgProfileURL != "" {
-		// Separate GH client for the profile refresh
-		orgProfileURL, err := url.Parse(cfg.Server.OrgProfileURL)
-		if err != nil {
-			log.Warn().Str("URL", cfg.Server.OrgProfileURL).Msg("Unable to parse url from string format")
-
+	if orgProfileLocation != "" {
+		// Check that the profile conforms to the expected format
+		location := strings.SplitN(orgProfileLocation, ":", 3)
+		if len(location) != 3 {
+			return fmt.Errorf("invalid organization profile location: %s", orgProfileLocation)
 		}
```

Comment:
Probably shouldn't block merge, but this should really use the one parsing function instead of having the format definition in 2 places.